components/ThemedImage.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import { useTheme } from 'next-themes';
+
+function ThemedImage(fileName: string | any) {
```

Comment:
Did you mean to include 'any' here? Using 'any' as a type in TypeScript, effectively disables type checking for this part of the code, negating the main benefits (type-checking) of using TypeScript.

Reply:
You're absolutely right. I'll take another look at this. Thanks :D